fsnotify v1.5.0 and GOOS=freebsd does not build

`GOOS=freebsd go` build works with Go 1.17 and fsnotify v1.4.9, but stops working with fsnotify v1.5.0.

The error message is:

```
vendor/github.com/fsnotify/fsnotify/kqueue.go:193:43: undefined: unix.O_SYMLINK
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -30,6 +30,7 @@ jobs:
 
   testFreeBSD:
     runs-on: macos-10.15
+    name: test (freebsd, 1.17)
     steps:
       - uses: actions/checkout@v2
       - name: test (freebsd, 1.17)

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -28,12 +28,26 @@ jobs:
         run: |
           go test --race ./...
 
+  testFreeBSD:
+    runs-on: macos-10.15
+    name: Test FreeBSD
+    steps:
+      - uses: actions/checkout@v2
+      - name: Test in FreeBSD
+        id: test
+        uses: vmactions/freebsd-vm@v0.1.5
+        with:
+          usesh: true
+          prepare: pkg install -y go
+          run: |
+            go test
+
   lint:
     runs-on: ubuntu-latest
     steps:
       - name: setup Go
         uses: actions/setup-go@v2
-        with: 
+        with:
           go-version: '1.17'
 
       - name: checkout
@@ -53,5 +67,3 @@ jobs:
         with:
           version: latest
           skip-go-installation: true
-          
-

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -30,10 +30,9 @@ jobs:
 
   testFreeBSD:
     runs-on: macos-10.15
-    name: Test FreeBSD
     steps:
       - uses: actions/checkout@v2
-      - name: Test in FreeBSD
+      - name: test (freebsd, 1.17)
         id: test
         uses: vmactions/freebsd-vm@v0.1.5
         with:

